Dockerfile

```diff
@@ -1,13 +1,9 @@
-FROM golang:alpine3.15
-
-WORKDIR /root/
-COPY ./go.mod ./go.sum ./
+FROM golang as build
```

Comment:
any particular reason to move away from golang:alpine3.15?

Reply:
Yes there is. Apparently I've tried to slim down binary by [ldflags](https://pkg.go.dev/cmd/go#hdr-Compile_packages_and_dependencies) and [trimpath](https://pkg.go.dev/cmd/go#hdr-Compile_packages_and_dependencies) which required g++ and non-alpine golang images comes without g++ so

Reply:
I'm not particularly attached to alpine, I just don't understand some of the changes you made and was hoping you'd explain or point me to some documentation on the subject.

Reply:
@vmorganp Can you point me what you didn't understand? I can write comments above each docker step if you want